app/src/features/projects/view/components/TreatmentSpatialUnits.tsx

```diff
@@ -186,10 +186,10 @@ const TreatmentSpatialUnits: React.FC<IProjectSpatialUnitsProps> = (props) => {
             <Button
               id={'open-layer-menu'}
               data-testid={'open-layer-menu'}
-              variant="text"
+              variant="outlined"
               color="primary"
-              title={'Filter Treaatments'}
-              aria-label={'Filter Treatments'}
+              title={'Filter treaments by year'}
```

Comment:
Typo, should says "treatments" not "treaments"

Reply:
Spelling mistakes resolved.